### ISSUE TITLE
Unable to deploy - TypeError: this.getCallerIdentity is not a function

### ISSUE THREAD

**Issue by BradenM** [newrelic/serverless-newrelic-lambda-layers#68]
Hi,

I am unable to deploy when using the latest version of `serverless-newrelic-lambda-layers` due to a `TypeError` being thrown:
```
Serverless: DOTENV: Loading environment variables from .env, .env.local, .env.dev:
Serverless:      - SLS_STAGE
Serverless:      - WS_CALLBACK_URL
Serverless:      - AWS_DAX_ENDPOINT
Serverless:      - RELIC_API_KEY
Serverless:      - RELIC_ACCOUNT_ID
Serverless:      - AWS_ACCESS_KEY_ID
Serverless:      - AWS_SECRET_ACCESS_KEY
Serverless:      - AWS_DEFAULT_REGION
Serverless:      - AWS_PROFILE
Loading Dynamic config...
Serverless: Warning: S3 Transfer Acceleration will not be enabled on deploymentBucket.
Serverless: WarmUp: no functions to warm up
Serverless: Using deployment bucket 'sls-bucket-awscc-dev'
Serverless: Bundling with Webpack...

-- WEBPACK BUNDLE OUTPUT -- 

Serverless: Package lock found - Using locked versions
Serverless: Packing external modules: axios@^0.20.0, aws-sdk@^2.750.0, amazon-dax-client@^1.2.4
Serverless: Plugins: ["serverless-webpack","serverless-plugin-warmup","serverless-dotenv-plugin","serverless-newrelic-lambda-layers","serverless-prune-plugin","serverless-dynamodb-local","serverless-domain-manager","serverless-deployment-bucket","serverless-localstack","serverless-stack-output","serverless-offline"]
Serverless: Adding NewRelic layer to awsConnect
Serverless: Adding NewRelic layer to connectionHandler
Serverless: Adding NewRelic layer to wsHandler
Serverless: Packaging service...
Serverless: Installing dependencies for custom CloudFormation resources...

  Type Error ---------------------------------------------

  TypeError: this.getCallerIdentity is not a function
      at NewRelicLambdaLayerPlugin.<anonymous> (/project/node_modules/serverless-newrelic-lambda-layers/dist/integration.js:28:43)
      at Generator.next (<anonymous>)
      at fulfilled (/project/node_modules/serverless-newrelic-lambda-layers/dist/integration.js:5:58)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com

  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              13.9.0
     Framework Version:         2.0.0 (local)
     Plugin Version:            4.0.2
     SDK Version:               2.3.1
     Components Version:        3.1.2

error Command failed with exit code 1.
```

Related Configuration:
```
plugins:
  - serverless-webpack
  - serverless-plugin-warmup
  - serverless-dotenv-plugin
  - serverless-newrelic-lambda-layers
  - serverless-prune-plugin
  - serverless-dynamodb-local
  - serverless-domain-manager
  - serverless-deployment-bucket
  - serverless-localstack
  - serverless-stack-output
  - serverless-offline

custom:
  stage: ${opt:stage, self:provider.stage}
  dynamic: ${file(config.js)}
  webpack:
    webpackConfig: ./webpack.config.js
    includeModules: true
    packager: 'yarn'
  newRelic:
    enableIntegration: true
    accountId: ${env:RELIC_ACCOUNT_ID}
    logEnabled: true
    cloudWatchFilter: '*'
    apiKey: ${env:RELIC_API_KEY}
```

<details>
<summary>Top Level Deps</summary>

```
  "devDependencies": {
    "@babel/core": "^7.11.6",
    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
    "@babel/plugin-transform-runtime": "^7.11.5",
    "@babel/preset-env": "^7.11.5",
    "axios-mock-adapter": "^1.18.2",
    "babel-eslint": "^10.1.0",
    "babel-jest": "^26.3.0",
    "babel-loader": "^8.1.0",
    "concurrently": "^5.3.0",
    "cross-env": "^7.0.2",
    "eslint": "^7.8.1",
    "eslint-config-airbnb-base": "^14.2.0",
    "eslint-config-prettier": "^6.10.1",
    "eslint-plugin-import": "^2.22.0",
    "eslint-plugin-jest": "^24.0.0",
    "eslint-plugin-prettier": "^3.1.4",
    "jest": "^26.4.2",
    "jest-date-mock": "^1.0.8",
    "prettier": "^2.1.1",
    "serverless": "^2.0.0",
    "serverless-deployment-bucket": "^1.1.2",
    "serverless-domain-manager": "^4.2.0",
    "serverless-dotenv-plugin": "^3.0.0",
    "serverless-dynamodb-local": "^0.2.39",
    "serverless-graph": "^0.3.2",
    "serverless-localstack": "^0.4.27",
    "serverless-newrelic-lambda-layers": "^0.2.0",
    "serverless-offline": "^6.7.0",
    "serverless-plugin-warmup": "^4.9.0",
    "serverless-prune-plugin": "^1.4.2",
    "serverless-sentry": "^2.0.2",
    "serverless-stack-output": "^0.2.3",
    "serverless-webpack": "^5.3.4",
    "shelljs": "^0.8.3",
    "thread-loader": "^2.1.3",
    "webpack": "^4.44.1",
    "webpack-node-externals": "^2.5.2"
  },
  "dependencies": {
    "amazon-dax-client": "^1.2.4",
    "aws-sdk": "^2.750.0",
    "axios": "^0.20.0",
    "babel-core": "^7.0.0-bridge.0"
  }


```

</details>

**Comment by kolanos** [comment]
@BradenM Thanks for reporting. We will get this fixed. In the meantime, you should downgrade to `v0.1.x`.

**Comment by kolanos** [comment]
@BradenM Ok, we've released a fix. Let us know if you see any other errors. Thanks again!

**Comment by BradenM** [comment]
@kolanos 

No luck after upgrading :grimacing: 

See my comment: https://github.com/newrelic/serverless-newrelic-lambda-layers/commit/01d8158102ece898e6ecdf9a3e7f2151bf37d237#r42239779

**Comment by kolanos** [comment]
@BradenM Should be fixed now with `v0.2.2`.

**Comment by BradenM** [comment]
:+1: Can confirm. Thanks!
